demos/devkit-simple-tutorial/src/views/Form.js

```diff
@@ -147,15 +147,20 @@ export class AppParticipateForm extends LitElement {
     const headers = {
       'Content-Type': 'application/json',
       'Access-Control-Allow-Origin': '*',
-      Authorization: `Bearer ${this._apiToken}`,
+      // Authorization: `Bearer ${this._apiToken}`,
     };
     await fetch(
-      'https://devkit-pce-staging.azurewebsites.net/v0/user-events/consent',
+      'https://devkit-pce-staging.azurewebsites.net/v0/user-events/consent/unsafe',
       {
         method: 'POST',
         headers,
         body: JSON.stringify({
           consentId: '28b5bee0-9db8-40ec-840e-64eafbfb9ddd',
+          app_id: '6f083c15-4ada-4671-a6d1-c671bc9105dc',
+          data_subject: {
+            id: localStorage.getItem('priv_user_id'),
```

Comment:
Why local storage? Can you use the value of the email input field?

Reply:
You're right, I just kept how it was before. Switched to get from the input